### ISSUE TITLE
Not an issue - just a question

### ISSUE THREAD

**Issue by millzee01** [saleh1312/football-players-matching#1]
Hi there,  gr8 work.

I'm doing something similar but I need to extract a frame that just contains the players and the ball in a png.

Do you think this is achievable with your code?

Many thanks

**Comment by saleh1312** [comment]
> Hi there, gr8 work.
> 
> I'm doing something similar but I need to extract a frame that just contains the players and the ball in a png.
> 
> Do you think this is achievable with your code?
> 
> Many thanks

yeah you can , some parts of my code will not be needed to your goal , just detect the players and the ball and check if there are players and ball in the frame
